tests/test_asgi.py

```diff
@@ -29,7 +29,7 @@ async def events() -> AsyncIterator[dict]:
 
 @pytest_asyncio.fixture
 async def client(app: ASGIApp) -> AsyncIterator[httpx.AsyncClient]:
-    async with httpx.AsyncClient(app=app) as client:
+    async with httpx.AsyncClient(transport=httpx.ASGITransport(app)) as client:
```

Comment:
CI says ASGIApp is not a type accepted by ASGITransport

I suppose we can type it as Any instead?